.azure-pipelines.yml

```diff
@@ -155,15 +155,18 @@ jobs:
 ########################################################################################
 - job:
   displayName: 'Windows'
-  condition: False
 
   pool:
     vmImage: 'vs2017-win2016'
 
   variables:
     CONDA_REQUIREMENTS: requirements.txt
     CONDA_REQUIREMENTS_DEV: requirements-dev.txt
-    CONDA_INSTALL_EXTRA: "codecov"
+    CONDA_INSTALL_EXTRA: "codecov gmt=6.0.0"
+    # ctypes.CDLL cannot find conda's libraries
+    GMT_LIBRARY_PATH: 'C:\Miniconda\envs\testing\Library\bin'
+    # Due to potential bug in GMT, we have to set GMT_SHAREDIR manually
+    GMT_SHAREDIR: 'C:\Miniconda\envs\testing\Library\share\gmt'
 
   strategy:
     matrix:
```

Comment:
There's a ghostscript v9.22 package for Windows release 9 Jul 2019 at https://anaconda.org/conda-forge/ghostscript/files. Could test that out and see if it works?

Reply:
Good to know that there is a Ghostscript for Windows. With that version, the failures for `logo` are gone.